SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -22,14 +22,16 @@
   <!-- **************************************************************************** -->
   <!-- Using tasks -->
   <!-- **************************************************************************** -->
-  <PropertyGroup Condition=" $(SonarQubeTempPath) != '' ">
+  <PropertyGroup Condition=" $(SonarQubeTempPath) != '' AND $(SonarQubeBuildTasksAssemblyFile) == '' ">
     <!-- Assume that the tasks assembly is in the same location as this targets file 
          or in a parent directory unless another location has already been specified. -->
-    <SonarQubeBuildTasksAssemblyFile Condition=" $(SonarQubeBuildTasksAssemblyFile) == '' ">$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), SonarQube.MSBuild.Tasks.dll))\SonarQube.MSBuild.Tasks.dll</SonarQubeBuildTasksAssemblyFile>
+    <SonarQubeBuildTasksAssemblyFile>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), SonarQube.MSBuild.Tasks.dll))SonarQube.MSBuild.Tasks.dll</SonarQubeBuildTasksAssemblyFile>
```

Comment:
https://msdn.microsoft.com/en-us/library/dd633440.aspx#BKMK_GetDirectoryNameOfFileAbove

I think the backlash might actually be missing